src/main/java/com/bgsoftware/superiorskyblock/island/SIsland.java

```diff
@@ -2933,9 +2940,16 @@ public BigDecimal getIslandLevel() {
     @Override
     public BigDecimal getRawLevel() {
         BigDecimal islandLevel = this.islandLevel.get();
+        RoundingMode roundingMode;
+
+        try {
+            roundingMode = RoundingMode.valueOf(plugin.getSettings().getIslandLevelRoundingMode());
+        } catch (IllegalArgumentException error) {
+            roundingMode = RoundingMode.HALF_UP;
+        }
```

Comment:
It's better that the SettingsContainer will already do this logic. and `getIslandLevelRoundingMode` would return RoundingMode